docs/examples/multimodal_classification.md

```diff
@@ -0,0 +1,314 @@
+This example shows how to build a multimodal classifier with Ludwig.
+
+If you'd like to run this example interactively in Colab, open one of these notebooks and try it out:
+
+- Ludwig CLI: [![Multimodal Classification with Ludwig CLI](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ludwig-ai/ludwig-docs/blob/daniel/twitter_bots_example/docs/examples/multimodal_classification/Multimodal_Classification_with_Ludwig_CLI.ipynb)
```

Comment:
just for the record will need to remember to change branch name before the merge.  first to `ludwig05` then again when merged to `master`.